goalie/go_mesh_seq.py

```diff
@@ -205,7 +233,8 @@ def indicate_errors(
                 for f in self.fields:
                     # Transfer solutions associated with the current field f
                     transfer(self.solutions[f][FWD][i][j], u[f])
-                    transfer(self.solutions[f][FWD_OLD][i][j], u_[f])
+                    if self.field_types[f] == "unsteady":
+                        transfer(self.solutions[f][FWD_OLD][i][j], u_[f])
                     transfer(self.solutions[f][ADJ][i][j], u_star[f])
                     transfer(self.solutions[f][ADJ_NEXT][i][j], u_star_next[f])
```

Comment:
Updating `forward` and `forward_old` here only serves to update the coefficients in the form. Since there is no lagged solution in the variational form for "steady" fields, I do not do that here despite that the notion of a "lagged solution" still exists here (i.e. the soln at the previous timestep).

But I kept this update for `adjoint_next` since in the next few lines we compute the average. I'm not familiar with the theory here so I don't know if it makes sense or not, so I just left it as it is.